res/controllers/Denon-Prime-4-scripts.js

```diff
@@ -186,6 +186,16 @@ Prime4.init = function(_id, _debug) {
     Prime4.mixerC = new mixerStrip(3, 2);
     Prime4.mixerD = new mixerStrip(4, 3);
 
+    // Load song to deck with library encoder button
+    // NOTE: This only works on Deck 1. I intend on making it load
+    //       to the first available deck in the future, but haven't
+    //       figured that behaviour out yet.
+    Prime4.encoderLoad = new components.Button({
+        midi: [0x9F, 0x06],
+        group: "[Channel1]",
+        key: "LoadSelectedTrack",
```

Comment:
I think you'll want `[Library]` `GoToItem`. That behaves sort of like the Enter Key on the keyboard. You can read more about it in the manual: https://manual.mixxx.org/2.3/en/chapters/appendix/mixxx_controls.html#control-[Library]-GoToItem
```suggestion
    Prime4.encoderLoad = new components.Button({
        midi: [0x9F, 0x06],
        group: "[Library]",
        key: "GoToItem",
```

Reply:
Brilliant, thank you.